src/components/feedback/style/feedback.less

```diff
@@ -15,6 +15,10 @@
   .progress {
     margin-top: 8px;
   }
+  textarea {
+    max-width: 320px;
+    max-height: 75px;
+  }
```

Comment:
That makes sense to block the width but use `max-width:100%;`.

I'm not sure about the height, when you have a long text it can beuseful.